example/large/multi/14-multi-language/build.mill

```diff
@@ -0,0 +1,50 @@
+package build
+import mill._, javascriptlib._, pythonlib._, javalib.{TestModule => JTestModule, _}
+
+object client extends ReactScriptsModule
+
+object `sentiment-analysis` extends PythonModule {
+  def mainScript = Task.Source { millSourcePath / "src" / "foo.py" }
+
+  def pythonDeps = Seq("Jinja2==3.1.4", "textblob==0.19.0")
+}
+
+object server extends JavaModule {
+  def ivyDeps = Agg(
+    ivy"org.springframework.boot:spring-boot-starter-web:2.5.6",
+    ivy"org.springframework.boot:spring-boot-starter-actuator:2.5.6"
+  )
+
+  /** Bundle client & sentiment-analysis as resource */
+  def resources = Task.Sources {
+    os.copy(client.bundle().path, Task.dest / "static")
+    os.makeDir.all(Task.dest / "analysis")
+    os.copy(`sentiment-analysis`.bundle().path, Task.dest / "analysis" / "analysis.pex")
+    super.resources() ++ Seq(PathRef(Task.dest))
+  }
+
+  object test extends JavaTests with JTestModule.Junit5 {
```

Comment:
Rather than using the rename, let's just use the qualified name `javalib.TestModule`